users/tests/test_models.py

```diff
@@ -53,3 +53,15 @@ def test_needs_vote_affirmation(self):
         )
 
         self.assertTrue(member3.needs_vote_affirmation)
+
+class UserGroupsModelsTestCase(TestCase):
+    def test_create_usergroup(self):
+        group = UserGroup.objects.create(
+            name='PLUG',
+            url='http://meetup.com/plug',
+            url_type=0
+        )
+        
+        self.assertNotEqual(group, None)
+        self.assertFalse(group.approved)
```

Comment:
Can you also test that the default value of `start_date` is `None`?

Reply:
Sure, done.